main.go

```diff
@@ -119,6 +125,36 @@ func run(p params) error {
 //go:embed htmlOutput.html.tpl
 var htmlOutputTemplate string
 
+func (j junit2jira) convertToSlack(tc []testCase) error {
```

Comment:
:thinking:
```suggestion
func (j junit2jira) maybeConvertToSlack(tc []testCase) error {
```

Reply:
Why `maybe`? I've never seen this prefix for a function that returns error?

Reply:
Because this function is a no-op if `j.slackOutput == ""`. Or perhaps move the condition to the call site and then the name is fine.

Reply:
let's name it `createSlackMessage` and have same convention as with other functions

https://github.com/stackrox/junit2jira/blob/d36b86edd823b074a7dbde0bc7a1f4b4620f0866/main.go#L157-L160

https://github.com/stackrox/junit2jira/blob/d36b86edd823b074a7dbde0bc7a1f4b4620f0866/main.go#L193-L196